app/services/proofing/lexis_nexis/proofer.rb

```diff
@@ -4,6 +4,7 @@ module Proofing
   module LexisNexis
     class Proofer < Proofing::Base
       Config = RedactedStruct.new(
+        :ddp_workflow,
         :instant_verify_workflow,
```

Comment:
I think we may want a separate config struct for DDP? We're going to have Org ID too, and a differnent Base url, right? It might be easier to make sure we don't mix up RDP and DDP?